lib/pg_sequencer/schema_dumper.rb

```diff
@@ -22,7 +22,7 @@ module SchemaDumper
     extend ActiveSupport::Concern
 
     included do
-      alias_method_chain :tables, :sequences
+      alias_method :tables, :sequences
```

Comment:
please read how to replace alias_method_chain 
http://www.justinweiss.com/articles/rails-5-module-number-prepend-and-the-end-of-alias-method-chain/